runtime/cere-dev/src/governance/tracks.rs

```diff
@@ -0,0 +1,267 @@
+const SUP_WHITELISTED_CALLER: Curve =
+	Curve::make_reciprocal(1, 28, percent(20), percent(5), percent(50));
+
+const TRACKS_DATA: [(u16, pallet_referenda::TrackInfo<Balance, BlockNumber>); 13] = [
+	(
+		0,
+		pallet_referenda::TrackInfo {
+			name: "root",
+			max_deciding: 1,
+			decision_deposit: 100 * GRAND,
+			prepare_period: 2 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 24 * HOURS,
+			min_enactment_period: 24 * HOURS,
+			min_approval: APP_ROOT,
+			min_support: SUP_ROOT,
+		},
+	),
+	(
+		1,
+		pallet_referenda::TrackInfo {
+			name: "whitelisted_caller",
+			max_deciding: 100,
+			decision_deposit: 10 * GRAND,
+			prepare_period: 30 * MINUTES,
+			decision_period: 28 * DAYS,
+			confirm_period: 10 * MINUTES,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_WHITELISTED_CALLER,
+			min_support: SUP_WHITELISTED_CALLER,
+		},
+	),
+	(
+		10,
+		pallet_referenda::TrackInfo {
+			name: "staking_admin",
+			max_deciding: 10,
+			decision_deposit: 5 * GRAND,
+			prepare_period: 2 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_STAKING_ADMIN,
+			min_support: SUP_STAKING_ADMIN,
+		},
+	),
+	(
+		11,
+		pallet_referenda::TrackInfo {
+			name: "treasurer",
+			max_deciding: 10,
+			decision_deposit: GRAND,
+			prepare_period: 2 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 24 * HOURS,
+			min_approval: APP_TREASURER,
+			min_support: SUP_TREASURER,
+		},
+	),
+	(
+		13,
+		pallet_referenda::TrackInfo {
+			name: "fellowship_admin",
+			max_deciding: 10,
+			decision_deposit: 5 * GRAND,
+			prepare_period: 2 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_FELLOWSHIP_ADMIN,
+			min_support: SUP_FELLOWSHIP_ADMIN,
+		},
+	),
+	(
+		14,
+		pallet_referenda::TrackInfo {
+			name: "general_admin",
+			max_deciding: 10,
+			decision_deposit: 5 * GRAND,
+			prepare_period: 2 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_GENERAL_ADMIN,
+			min_support: SUP_GENERAL_ADMIN,
+		},
+	),
+	(
+		20,
+		pallet_referenda::TrackInfo {
+			name: "referendum_canceller",
+			max_deciding: 1_000,
+			decision_deposit: 10 * GRAND,
+			prepare_period: 2 * HOURS,
+			decision_period: 7 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_REFERENDUM_CANCELLER,
+			min_support: SUP_REFERENDUM_CANCELLER,
+		},
+	),
+	(
+		21,
+		pallet_referenda::TrackInfo {
+			name: "referendum_killer",
+			max_deciding: 1_000,
+			decision_deposit: 50 * GRAND,
+			prepare_period: 2 * HOURS,
+			decision_period: 28 * DAYS,
+			confirm_period: 3 * HOURS,
+			min_enactment_period: 10 * MINUTES,
+			min_approval: APP_REFERENDUM_KILLER,
+			min_support: SUP_REFERENDUM_KILLER,
+		},
+	),
+	(
+		30,
```

Comment:
Is there some logic behind choosing the IDs? Why doesn't it go one by one?

Reply:
There is no in-code logic as far as I know. The order of the IDs is just a conventions that depends on Track's severity as far as I know. I just [copied the Polkadot's tracks](https://github.com/paritytech/polkadot-sdk/blob/release-polkadot-v1.1.0/polkadot/runtime/polkadot/src/governance/tracks.rs#L68) excluding ones that are related to parachains.